packages/errors/src/errors.ts

```diff
@@ -279,7 +279,23 @@ export const AllCypressErrors = {
 
         https://on.cypress.io/parallel-disallowed`
   },
-  CLOUD_PARALLEL_GROUP_PARAMS_MISMATCH: (arg1: {runUrl: string, parameters: any}) => {
+  CLOUD_PARALLEL_GROUP_PARAMS_MISMATCH: (arg1: {runUrl: string, parameters: any, payload: any }) => {
+    let params: any = arg1.parameters
+
+    if (arg1.payload?.differentParams) {
```

Comment:
when would `differentParams` not exist if there is a mismatch?

Reply:
This is more a safeguard in case the cloud changes were released after the app